LCI connect and disconnect (#1364)
harness/developer-hub: docs/chaos-engineering/configure-chaos-experiments/chaos-infrastructure/connect-disconnect-linux-infrastructure.md

---
title: Using Linux infrastructure to execute chaos faults
sidebar_position: 4
---
## Introduction
This section describes how HCE can be used to inject chaos into Linux machines on the Linux Chaos Infrastructure (LCI). Connecting to LCI, creating experiments, running them, and then disconnecting from LCI is similar to performing the same steps on [Kubernetes infrastructure](./connect-chaos-infrastructures).

## Steps to inject chaos into LCI
1. Connect to LCI,
2. Configure chaos on LCI and deploy it,
3. Add faults to the deployed LCI,
4. Run the experiment.

Described below are the detailed steps to inject chaos and run experiments on LCI.
### 1. Connect to LCI
* To connect to LCI, create a new environment, and specify the details of the environment. Select **Create**.

![](./static/linux-user-guide/1.new-env.png)

* Select **Linux** as the **Infrastructure type**, and select **Enable chaos**.

![](./static/linux-user-guide/2.select-linux.png)

### 2. Configure chaos on LCI and deploy it
* Specify details to enable chaos on LCI, and select **Next**.

![](./static/linux-user-guide/3.configure-chaos.png)

> * Copy the command, select **Done**, and execute the copied command on your terminal.

Reviewer: ```suggestion
* Copy the command, select **Done**, and execute the copied command in your Linux machine.
```
